### PR TITLE
Fix style issues after integration

### PR DESCRIPTION
**Pull request by kulcsaradam**

### CODE REVIEW

File: src/jit/Backend.cpp [hunk 1]
```diff
@@ -41,8 +41,7 @@ extern "C" {
 #error Unsupported architecture
 #endif
 
-#define OffsetOfContextField(field) \
-    (static_cast<sljit_sw>(offsetof(ExecutionContext, field)) - static_cast<sljit_sw>(sizeof(ExecutionContext)))
+#define OffsetOfContextField(field) (static_cast<sljit_sw>(offsetof(ExecutionContext, field)) - static_cast<sljit_sw>(sizeof(ExecutionContext)))
```
Comment (zherczeg) [inline src/jit/Backend.cpp:44]: Should keep the original

File: src/jit/Backend.cpp [hunk 2]
```diff
@@ -203,11 +198,9 @@ static void operandToArg(Operand* operand, JITArg& arg)
     }
 }
 
-#define GET_TARGET_REG(arg, default_reg) \
-    (((arg)&SLJIT_MEM) ? (default_reg) : (arg))
+#define GET_TARGET_REG(arg, default_reg) (((arg)&SLJIT_MEM) ? (default_reg) : (arg))
```
Comment (zherczeg) [inline src/jit/Backend.cpp:201]: I would not change the defines.

File: src/jit/FloatMathInl.h [hunk 3]
```diff
@@ -47,36 +44,25 @@ static void floatOperandToArg(sljit_compiler* compiler,
...
     }
 }
 
 // Float operations.
 // TODO Canonical NaN
-static sljit_f32 floatNearest(sljit_f32 val)
-{
-    return std::nearbyint(val);
-}
+static sljit_f32 floatNearest(sljit_f32 val) { return std::nearbyint(val); }
```
Comment (zherczeg) [inline src/jit/FloatMathInl.h:63]: I think the original form is preferred for these

**Review by zherczeg — CHANGES_REQUESTED** [review]
Looks better! I would keep the defines and one-statement functions

File: src/jit/Backend.cpp [hunk 4]
```diff
@@ -206,8 +202,7 @@ static void operandToArg(Operand* operand, JITArg& arg)
 #define GET_TARGET_REG(arg, default_reg) \
     (((arg)&SLJIT_MEM) ? (default_reg) : (arg))
 #define IS_SOURCE_REG(arg) (!((arg) & (SLJIT_MEM | SLJIT_IMM)))
-#define GET_SOURCE_REG(arg, default_reg) \
-    (IS_SOURCE_REG(arg) ? (arg) : (default_reg))
+#define GET_SOURCE_REG(arg, default_reg) (IS_SOURCE_REG(arg) ? (arg) : (default_reg))
```
Comment (zherczeg) [inline src/jit/Backend.cpp:205]: I would keep the original

File: src/jit/FloatMathInl.h [hunk 5]
```diff
@@ -122,15 +114,9 @@ static sljit_f64 floatMax(sljit_f64 lhs, sljit_f64 rhs)
...
-{
-    return std::copysign(lhs, rhs);
-}
+static sljit_f32 floatCopySign(sljit_f32 lhs, sljit_f32 rhs) { return std::copysign(lhs, rhs); }
 
-static sljit_f64 floatCopySign(sljit_f64 lhs, sljit_f64 rhs)
-{
-    return std::copysign(lhs, rhs);
-}
+static sljit_f64 floatCopySign(sljit_f64 lhs, sljit_f64 rhs) { return std::copysign(lhs, rhs); }
```
Comment (zherczeg) [inline src/jit/FloatMathInl.h:119]: I would keep these two

File: src/jit/InstList.cpp [hunk 6]
```diff
@@ -37,54 +37,27 @@ Index Instruction::resultCount()
...
-{
-    delete[] m_targetLabels;
-}
+BrTableInstruction::~BrTableInstruction() { delete[] m_targetLabels; }
 
-ComplexInstruction::~ComplexInstruction()
-{
-    delete[] params();
-}
+ComplexInstruction::~ComplexInstruction() { delete[] params(); }
```
Comment (zherczeg) [inline src/jit/InstList.cpp:42]: I would keep these two as well, one liner functions should be merged only in classes for getters / setter.

File: src/runtime/Module.h [hunk 7]
```diff
@@ -430,4 +430,4 @@ class Module : public Object {
 
 } // namespace Walrus
 
-#endif // __WalrusModule__
+#endif // __WalrusModule__
```
Comment (zherczeg) [inline src/runtime/Module.h:433]: Don't remove this line

**Review by zherczeg — CHANGES_REQUESTED** [review]
Quite good! The main problem is merging single line functions.

**Comment by clover2123** [comment]
[FYI] In `interp` branch, we use clang-format-10 for code format checker.
We run the following command to check the code format
```sh
tools/check_tidy.py
```

**Comment by zherczeg** [comment]
Unfortunately it does not check line breaks and invalid variable namings :(

File: src/jit/Locals.cpp [hunk 8]
```diff
@@ -72,10 +72,7 @@ struct LocalRange {
     {
     }
 
-    bool operator<(const LocalRange& other) const
-    {
-        return (start < other.start);
-    }
+    bool operator<(const LocalRange& other) const { return (start < other.start); }
```
Comment (zherczeg) [inline src/jit/Locals.cpp:75]: Revert this.

**Review by zherczeg — CHANGES_REQUESTED** [review]
Good progress. Only one change remained.

File: src/jit/Locals.cpp (hunk 8 above)
Comment (kulcsaradam) [reply src/jit/Locals.cpp]: It should be done.

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**